lib/fileutils.rb

```diff
@@ -196,6 +196,8 @@ def self.private_module_function(name)   #:nodoc:
   #
   #   FileUtils.pwd # => "/rdoc/fileutils"
   #
+  # Related: FileUtils.cd.
+  #
   # FileUtils.getwd is an alias for FileUtils.pwd.
```

Comment:
I think we should put the alias section before the related section. I think the alias section is more relevant to the current method and is more important to it deserves to be higher.

Reply:
Once again, I have mis-remembered the doc guide:

> The general structure of the method documentation should be:
- Calling sequence (for methods written in C).
- Synopsis (short description).
- Details and examples.
- Argument description (if necessary).
- Corner cases and exceptions.
- Aliases.
- Related methods (optional).

Will fix.

Reply:
Fixed.